### DISCUSSION TITLE
Free disk space

### DISCUSSION THREAD

**Discussion by mpp-olano** [datalust/seq-tickets#1381] [Help]
Is there any reason why free disk checking is disabled in Seq2021.3 and How can it be enabled?

I have 3 instances of Seq within my environment running in containers on a VM which was recently upgraded from 2021.2 to 2021.3. It shows the Free Disk Space on one(see below):
![image](https://user-images.githubusercontent.com/87758886/142285928-24be6c29-f277-4b73-8a55-7a6ecf09dd33.png)

but doesn't show on the other two(see below)
![image](https://user-images.githubusercontent.com/87758886/142225109-a99755e1-faae-4903-9ab1-91910ec6310c.png)

Here the block in my Seq.json file in _data/seq_
  },
    "disableFreeDiskSpaceChecking": false
  },

**Comment by nblumhardt** [comment]
Hi @mpp-olano! Thanks for dropping by.

This will happen if the filesystem takes too long to return free disk space information. If your `/data` volume is mounted to a networked filesystem, for example, free space checking will sometimes require enumerating every file in every directory, which has a severe performance impact.

Does that sound likely to be the cause here? You should find some additional info in _Download a full diagnostic report_, also on that screen.

**Reply by mpp-olano** [reply]
Thanks for the response @nblumhardt . The volume is mounted locally not to a network filesystem. There existing the same configuration across all the 3 instances.
See screenshot below: 
![image](https://user-images.githubusercontent.com/87758886/142390503-795ce2e4-4d66-488f-9650-624f22f4f5a9.png)

**Reply by nblumhardt** [reply]
Interesting - thanks for the follow-up. I wonder if the local filesystem is slow, or some aspect of its projection through Docker?

If you restart the container, does the message appear again (reasonably) immediately?

We can take a look at internal diagnostics, too, if you're able to send the report through to `support@`. Thanks!
